src/els_code_navigation.erl

```diff
@@ -36,9 +36,13 @@ goto_definition( Uri
                       Kind =:= implicit_fun;
                       Kind =:= export_entry ->
   find(Uri, function, {F, A});
-goto_definition(_Uri, #{ kind := behaviour, id := Behaviour }) ->
-  case els_utils:find_module(Behaviour) of
-    {ok, Uri}      -> find(Uri, module, Behaviour);
+goto_definition( _Uri
+               , #{ kind := Kind, id := Module }
+               ) when Kind =:= atom;
+                      Kind =:= behaviour;
+                      Kind =:= module ->
```

Comment:
Does it make sense to add `module` here? Won't this be the case where the user is trying to get the definition of the module from the `-module(foo).` attribute?